GetServices/handler.ts

```diff
@@ -44,7 +44,7 @@ export function GetServicesHandler(
   serviceModel: ServiceModel
 ): IGetServicesHandler {
   return async (_, __) => {
-    const allRetrievedServicesIterator = await serviceModel.getCollectionIterator();
+    const allRetrievedServicesIterator = serviceModel.getCollectionIterator();
```

Comment:
@cloudify: we have to replicate all the following logic?

Reply:
what do you mean?

Reply:
In this function we get all services from cosmos and then:
- reduce results by removing "old" versions of the same service
- return an array representing an iterator "page".
But I' don't understand if, at client side, we can request next pages. Or even if we have to return all the service' s collection in a single page, it makes no sense to work with iterator.

Reply:
what do you mean with "it makes no sense to work with iterator" ?

Reply:
I mean that if we don't serve paginated results (if it's true), we can work directly with an array containing all results